src/Admin.php

```diff
@@ -167,10 +177,16 @@ public static function post_action_label($post_id) {
 
     $data['attributes'] = $attributes;
 
-    $qr_code = static::getProductQrCode($post_id, static::QR_CODE_SIZE);
+    if ($labelFormat[1] == 'landscape') {
+      $qr_code = static::getProductQrCode($post_id, 600);
```

Comment:
Please move this into a constant to stay consistent with the existing code:
```suggestion
      $qr_code = static::getProductQrCode($post_id, static::QR_CODE_SIZE_LANDSCAPE);
```